egui/src/context.rs

```diff
@@ -889,6 +889,16 @@ impl Context {
         self.memory().layer_id_at(pos, resize_grab_radius_side)
     }
 
+    /// Top-most layer.
```

Comment:
I would like to see an explanation, what is the top-most layer instead of this useless doc. The doc should answer to question when I should use it? What is its features?

Reply:
What about this?

```
Returns the id of the top-most area. When you click on or interact with an area, it will be moved above all other areas.
```

My personal use case is synchronizing the selected item in a list of open windows with the window that last got focus. This is a very specific use case, but it wasn't possible to do before.

Reply:
I pushed a commit changing it to

> The overall top-most layer. When an area is clicked on or interacted with, it is moved above all other areas.